katas/content/superposition/even_odd/solution.md

```diff
@@ -0,0 +1,21 @@
+### Solution
+
+Let’s look at some examples of basis states to illustrate the binary numbering system. 
+
+The 4 basis states on $N = 2$ qubits can be split in two columns, where the left column represents the basis states that form the required superposition state for `isEven = false` and the right column - the basis states that form the required superposition state for `isEven = true`.
```

Comment:
This explanation doesn't make much sense without the images... Maybe we can draw a table, with two columns: even (00 and 10) and odd (01 and 11), and instead of red frames use bold for the first N-1 binary digits (and another table for N=3), and rephrase accordingly